projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdfActivity.java

```diff
@@ -0,0 +1,327 @@
+    private static String lawsuitTemplateFilePath = "";
+    private static String lawsuitOutputPath = "";
+    private static String lawsuitOutputPdfFilePath = "";                             // Init when user confirms form.
+
+    // Date & Time formats: (Used for files names)
+    // TODO: Check if this should be as part of a "File manager" class.
+    private static final String DATE_TIME_FORMAT = "dd-M-yyyy hh-mm-ss";
+    private static final String DATE_FORMAT = "dd-M-yyyy";
+    private static final String TIME_ZONE = "Asia/Jerusalem";
+    private static SimpleDateFormat dateAndTimeFormatter;
+    private static SimpleDateFormat dateFormatter;
+
+    // Template's inner text (been read on create from template's file):
+    private static String templateContent = "";
+
+// ------------------------------- Lawsuit Form Fields ------------------------------- //
+    private String spamType = "הודעה אלקטרונית";        // Type: SMS/Phone call/Email.
+    private String claimAmount = "סכום תביעה";
+    private Date receivedSpamDate;                    // Date received the spam.
+    private Date currentDate;                         // Current date of lawsuit creation (used for lawsuit signature).
+    private boolean sentHaser = true;                 // Indicates whether user has sent.
+    private boolean sentMoreThanFiveLawsuits = false; // Indicates whether user has sent more then 5 lawsuits already in the past year.
+    private String moreThanFiveLawsuits ="הגיש";
+    private String lessThanFiveLawsuits = "לא הגיש/ו";
+    private String fiveLawsuitsStatus ="";            // Init onCreate().
+    private String claimCaseHaser = "למרות שח\"מ לא נתן את הסכמתו המפורשת מראש לקבלת דבר/י הפרסומת";
+    private String claimCaseSubscription = "למרות שח\"מ לא נתן את הסכמתו המפורשת מראש לקבלת דבר/י הפרסומת";
+    private String claimCase = "";                    // "Didn't subscribe" / "Sent 'haser' but got spam". Set on fillInTemplate().
+
+    // User:
+    private String userPrivateName = "שם פרטי";
+    private String userLastname = "שם משפחה";
+    private String userID = "ת ז משתמש";
+    private String userAddress = "כתובת משתמש";
+    private String userPhone = "טלפון משתמש";
+    private String userFax = "פקס משתמש";
+
+    // First spam company's details:
+    private String companyName = "שם החברה";
+    private String companyId = "מספר ח.פ";
+    private String companyAddress = "כתובת החברה";
+    private String companyPhone = "טלפון החברה";
+    private String companyFax = "פקס חברה";
+
+    // Second spam company's det ails:
+    private String company2Name = "שם החברה 2";
+    private String company2Id = "מספר ח.פ 2";
+    private String company2Address = "כתובת החברה 2";
+    private String company2Phone = "טלפון החברה 2";
+    private String company2Fax = "פקס חברה 2";
+// ----------------------------- - -------------------------------------------------------:
+    Button button_create_pdf;
```

Comment:
We use lowerCamelCase for member names, so `buttonCreatePdf`.
I would rename to `createPdfButton` to make it sound more like English.

Reply:
Done.